README.md

```diff
@@ -293,156 +195,34 @@ Your PDS member (or PS) is uploaded.
-
-1. Navigate to the explorer tree.
-2. Open the **Unix System Services (USS)** bar.
-3. Select a file you want to remove.
-4. Select the **Delete** button and press yes in the confirmation dropdown.
-   The file is deleted.
-
-#### Delete a directory
-
-1. Navigate to the explorer tree.
-2. Open the **Unix System Services (USS)** bar.
-3. Select a directory you want to remove.
-4. Select the **Delete** button and press yes in the confirmation dropdown.
-   The directory and all child files and directories are deleted.
-
-![Create and Delete](docs/images/ZE-CreateDelete.gif?raw=true "Create and Delete")
-<br /><br />
-
-#### View and access multiple USS profiles simultaneously
-
-1. Navigate to the explorer tree.
-2. Open the **Unix System Services (USS)** bar.
-3. Click the **Add Session** button on the right of the **Unix System Services (USS)** explorer bar.
-4. Select the profile that you want to add to the view as illustrated by the following screen.
-
-![View Profiles](docs/images/ZE-profile2.gif?raw=true "View Profiles")
-<br /><br />
-
----
-
-### JOBS
-
-You can use the following functionalities when interacting with Jobs:
-
-* **View a job**: You can view multiple jobs simultaneously.
-* **Download spool content**: You can download spool content on your computer.
-
-#### View a job
-
-1. Navigate to the explorer tree.
-2. Open the **JOBS** bar.
-3. Select a directory with JCL files.
-4. Right-click on the JCL you want to view, and click **Get JCL**.
-
-![View JOB](docs/images/ZE-jobs-get-jcl.gif?raw=true "View JOB")
-<br /><br />
+## Usage tips
 
-#### Download spool content
+* Use the **Add Favorite** feature to permanently store chosen data sets, USS files, and jobs in the **Favorites** folder. Right-click on a data set, USS file or jobs and select **Add Favorite**.
 
-1. Navigate to the explorer tree.
-2. Open the **JOBS** bar.
-3. Select a directory with JCL files.
-4. Click the **Download** icon next to a folder with the spool content.
-5. Save the file on your computer.
+* **Syntax Highlighting:** Zowe Explorer supports syntax highlighting for data sets. Fox example, you can use such extensions as [COBOL Language Support](https://marketplace.visualstudio.com/items?itemName=broadcomMFD.cobol-language-support) or [HLASM Language Support](https://marketplace.visualstudio.com/items?itemName=broadcomMFD.hlasm-language-support).
 
-![Download Spool](docs/images/ZE-jobs-download-spool.gif?raw=true "Download Spool")
-<br /><br />
+* **Edit a profile**: Click the **pencil** icon next to the **magnifying glass** icon in the explorer tree, and modify the information inside your profile.
 
----
+* **Delete a profile**: Right-click a chosen profile and select **Delete Profile** to permanently delete the profile. The functionality deletes a profile from your `.zowe` folder.
 
-### Extras
+* **Hide a profile**: You can hide a profile from the profile tree by right-clicking the profile and selecting the **Hide Profile** option.
```

Comment:
Just one addition here:
"If you want to add back a profile that you have previously hidden, select the + button to add a new profile and your hidden profile appears in the drop-down list."
Something like that.